packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -343,6 +343,10 @@ module.exports = {
       navigateFallbackWhitelist: [/^(?!\/__).*/],
       // Don't precache sourcemaps (they're large) and build asset manifest:
       staticFileGlobsIgnorePatterns: [/\.map$/, /asset-manifest\.json$/],
+      // Precache files in public/, and allow them to be accessed directly
+      staticFileGlobs: ['./public/**/*'],
```

Comment:
What is it resolving this path against?

Reply:
i've swapped this out for `paths.appPublic` in the latest commit